### ISSUE TITLE
[REQ] Diablo Torrent

### ISSUE THREAD

**Issue by RoloSoze** [Jackett/Jackett#1033]
**Indexer Name**
Diablo Torrent

**Type**
Movies, TV, Audio, XXX, Programs (PC/Mobile/Mac), Books, Console, Other

**Indexer URL**
https://diablotorrent.net/

**Indexer Language**
Hungarian

**Invite**
Sent

**Note**
Diablo Torrent is a very large general Hungarian tracker with almost 200K torrents

**Comment by kaso17** [comment]
invite received

**Comment by kaso17** [comment]
Added with 4453edd441756c9a30d2b1d01f139bd2f4c228d7
Is it possible to login from multiple PCs/browsers at the same time?
Do they have some kind of freeleech or multi upload (couldn't find any torrents)?

**Comment by armani316** [comment]
They don't have any freeleech or multi upload. I just tried logging in from 2 different browsers simultaneously on the same PC and it works fine. Not sure about logging in from multiple PCs. What's the significance of this aspect for Jackett?

**Comment by kaso17** [comment]
If I login in firefox it kills my session in chrome and vice versa. login via jackett (same PC) kills the sessions too. I couldn't find a profile option to change that behavior.

**Comment by armani316** [comment]
@kaso17 You're right. I just checked again, and the website kills my session if I log in from another browser, or Jackett. I had to refresh the 1st session to realize that I'd been logged out. I guess they don't allow multiple concurrent sessions.

**Comment by armani316** [comment]
@kaso17 Also, I just checked with friends. Diablo Torrents is actually ratio-free! I didn't know that myself! Could you please reflect this fact in Jackett?

**Comment by kaso17** [comment]
fixed with 95d50a608cca0d05033147ca53220f483992206d

**Comment by garfield69** [comment]
https://diablotorrent.net/browse?category[]=49&category[]=21&category[]=101&category[]=102&category[]=103&category[]=104&category[]=105&category[]=106&category[]=107&category[]=108&category[]=109&category[]=110&category[]=111&category[]=112&category[]=12&category[]=14&category[]=39&category[]=40&category[]=8&category[]=9&category[]=37&category[]=38&category[]=16&category[]=24&category[]=53&category[]=27&category[]=29&category[]=20&category[]=52&s_alapjan=7&s_sorrend=desc > 502 Bad Gateway nginx/1.19.3

**Comment by ilike2burnthing** [comment]
back up

**Comment by garfield69** [comment]
online
